src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -2106,4 +2106,9 @@ public static void setWriteDelay(int writeDelayInSeconds) {
     public static int getWriteDelay() {
         return conf.write_delay_in_s;
     }
+
+    public static boolean getIsNewCluster()
+    {
+        return Boolean.getBoolean("palantir_cassandra.is_new_cluster");
+    }
```

Comment:
StorageService already has an equivalent `isNewCluster` function. Can we somehow merge those two?

Reply:
done

Reply:
My preference is to put all of these flags in DatabaseDescriptor, since that seems be most aligned with the existing code

Reply:
[ed21485](https://github.com/palantir/cassandra/pull/647/commits/ed2148501f0f03e64ec49da3d5c2acb6a3670111)